### PR TITLE
TLS 1.0 and 1.1 deprecated in all modern browsers

### PR DESCRIPTION
**Pull request by daytonlowell**
Links to all sources are included in the links section of the JSON.

### CODE REVIEW

File: features-json/tls1-1.json [hunk 1]
```diff
@@ -317,7 +333,7 @@
       "7.12":"y"
     }
   },
-  "notes":"",
+  "notes":"TLS 1.0 & 1.2 are deprecated in Chrome, Edge, Firefox, Internet Explorer 11, & Safari. Support will be removed in March 2020.",
```
Comment (Schweinepriester) [inline features-json/tls1-1.json]: It should be "TLS 1.0 & 1.1", shouldn't it? 0:-)

```suggestion
  "notes":"TLS 1.0 & 1.1 are deprecated in Chrome, Edge, Firefox, Internet Explorer 11, & Safari. Support will be removed in March 2020.",
```

**Comment by daytonlowell** [comment]
Yup, thanks. Fat fingered it, I guess.

**Comment by Fyrd** [comment]
Thanks!

**Comment by daytonlowell** [comment]
Oh also, I noticed that TLS 1.0 on the site says “This network security protocol is supported in effectively all browsers (since IE6+, Firefox 2+, Chrome 1+ etc)”.

That should probably be changed to say deprecated instead of supported, but I don’t think it’s in the JSON. I couldn’t find it, anyway.

**Pull request merged**